client/components/forms/form-select/index.jsx

```diff
@@ -12,13 +12,15 @@ const FormSelect = React.createClass( {
 	},
 
 	render() {
-		const { className, isError, ...props } = this.props,
-			classes = classNames( className, 'form-select', {
-				'is-error': isError,
-			} );
+		const { inputRef, className, isError, ...props } = this.props;
+		const classes = classNames( className, 'form-select', {
+			'is-error': isError,
+		} );
+
+		const ref = inputRef ? { ref: inputRef } : {};
 
 		return (
-			<select { ...props } className={ classes }>
+			<select { ...props } { ...ref } className={ classes }>
```

Comment:
I'm not bugged, just wondering if we need to go through the special effort of formatting a `ref` object if it instead works just as well to pass along verbatim if React will ignore it as `undefined` anyways.

Which it is apparently guaranteed to do so: https://github.com/facebook/react/blob/7dc27d3/src/isomorphic/classic/element/__tests__/ReactElement-test.js#L191-L204